### PR TITLE
Change the default imagepullpolicy for performance test

### PR DESCRIPTION
**Pull request by Avilir**
The default image pull policy is : Always
Which mean that in the pod attach / reattach we are measuring the pull image time in the total attach time.

This PR change the default image pull policy to : IfNotPresent
So, the image pull time will no calculate in the attach time, since we are pulling it separately in the beginning of the test and not as part as the pod creation process.

This PR also remove some code duplication from the pod_reattach test.

the Pulling image is also implements in the bulk_pod_attach test
some duplicated code remove from the bulk_pod_attach test.

### CODE REVIEW

**Review by keesturam — APPROVED** [review]
LGTM Avi.

File: tests/e2e/performance/csi_tests/test_bulk_pod_attachtime_performance.py [hunk 1]
```diff
@@ -78,19 +67,15 @@ def base_setup(self, project_factory, interface_type, storageclass_factory):
...
                 *[constants.CEPHBLOCKPOOL, 240],
-                marks=[pytest.mark.performance],
             ),
             pytest.param(
                 *[constants.CEPHFILESYSTEM, 120],
-                marks=[pytest.mark.performance],
             ),
             pytest.param(
                 *[constants.CEPHFILESYSTEM, 240],
-                marks=[pytest.mark.performance],
```
Comment (ypersky1980) [inline tests/e2e/performance/csi_tests/test_bulk_pod_attachtime_performance.py:93]: Avi, why did you remove the mark ?

File: tests/e2e/performance/csi_tests/test_pod_reattachtime.py [hunk 2]
```diff
@@ -32,25 +30,6 @@ def setup(self):
...
-                },
-            }
-        }
-        # during development use the dev ES so the data in the Production ES will be clean.
-        if self.dev_mode:
-            self.crd_data["spec"]["elasticsearch"] = {
-                "server": config.PERF.get("dev_es_server"),
-                "port": config.PERF.get("dev_es_port"),
-                "url": f"http://{config.PERF.get('dev_es_server')}:{config.PERF.get('dev_es_port')}",
-            }
```
Comment (ypersky1980) [inline tests/e2e/performance/csi_tests/test_pod_reattachtime.py:33]: Also  - why did you remove this section in both tests? Don't we need it to upload to the ES?

File: tests/e2e/performance/csi_tests/test_bulk_pod_attachtime_performance.py (hunk 1 above)
Comment (Avilir) [reply tests/e2e/performance/csi_tests/test_bulk_pod_attachtime_performance.py:93]: since we have a global performance marker on the class level, so we don't need this marker per test.

File: tests/e2e/performance/csi_tests/test_pod_reattachtime.py (hunk 2 above)
Comment (Avilir) [reply tests/e2e/performance/csi_tests/test_pod_reattachtime.py:33]: we have it as global assignment in the PASTest object setup.

**Review by ypersky1980 — APPROVED** [review]
Thanks for a prompt fix!
